### DISCUSSION TITLE
Is there a way to change user avatar on  gdm？

### DISCUSSION THREAD

**Discussion by Meow-2** [realmazharhussain/gdm-settings#254] [Q&A] [answered]
I set my avatar in ~/.face file refer to https://help.gnome.org/admin/gdm/stable/overview.html.en#facebrowser, but it does not work.Then I found the user avatar show on gdm is always from the gtk icon theme I set in gdm settings.How can I use my custom image as avatar.

Environment:
```
Arch Linux 6.6.44-1-lts
WM:dwm 6.3
DM:gdm 46.2-2
gdm-settings 4.4.2
```

**Comment by realmazharhussain** [comment, marked as answer]
You can modify the icon theme, or you can change your user's profile picture.

To change your user's profile picture on GNOME 46;

1. Open **Settings** app.
2. Click on the "System" section in the sidebar.
3. Click on "Users" item in the list
4. Click on the pencil icon on your profile picture.
5. Select your new profile picture.
